utils/index.js

```diff
@@ -1 +1,2 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
This rule is disabled for now, because we will have some more util's files. :smiley: